go.mod

```diff
@@ -1,11 +1,12 @@
 module go.viam.com/rdk
 
-go 1.23.8
```

Comment:
I see this is from using supercronic v0.2.34. Which was incidentally only released 3 days ago. v0.2.33 was on go 1.23.0.

I suspect this will cause a myriad of linter issues/auto upgrades of the go binary under the hood for people. `app` is using version v0.2.26.

I'd like to avoid potential problems due to a Go upgrade (admittedly, I don't understand 100% the ins and outs of why we've seen what we've seen). I assume it wouldn't be a problem to downgrade our usage of supercronic to v0.2.33?